About a compilation error

I came across this project in github and want to run and learn from it. But I got a compilation error: 
```
qemu-system-riscv64 \
	      \
	     -M virt,aia=aplic-imsic,aia-guests=4 -cpu rv64,x-aia=true -smp 1 -m 4096 \
	     -nographic \
	     -kernel target/riscv64gc-unknown-none-elf/release/salus \
	     -device guest-loader,kernel=tellus_guestvm,addr=0xc0200000
qemu-system-riscv64: -device guest-loader,kernel=tellus_guestvm,addr=0xc0200000: qemu_fdt_add_subnode: Failed to create subnode /chosen/module@0xc0200000: FDT_ERR_NOSPACE
Makefile:52: recipe for target 'run_tellus' failed
make: *** [run_tellus] Error 1

```
I have configured the environment as described in the README. But I still got a compilation error. I would like to know if the compilation fails because my environment is different.  
  
My qemu is version 7.0 , my rust version is `rustup 1.25.1 (bb60b1e89 2022-07-12)` and I run this project on Ubuntu 18.04. 

May I ask some solutions about this?

--- a/README.md
+++ b/README.md
@@ -21,6 +21,7 @@ There is a `Makefile` provided with targets for running various hosts in qemu.
 - qemu-system-riscv64 - build using  qemu
   [instructions](https://wiki.qemu.org/Hosts/Linux) with
   `--target-list=riscv64-softmmu`
+  - Note qemu 7.0.0 has a bug emulating riscv and a newer version from git must be used.
 
 ## Test VM
 

